zipline/algorithm.py

```diff
@@ -237,7 +238,9 @@ def __init__(self, *args, **kwargs):
                 data_frequency=self.data_frequency,
                 asset_finder=self.asset_finder,
                 slippage_func=VolumeShareSlippage(),
-                commission=PerShare()
+                commission=PerShare(),
+                # Default to NeverCancel in zipline
+                cancel_policy=kwargs.pop('cancel_policy', NeverCancel())
```

Comment:
what happened to the API method you had introduced in a previous version of this code?